src/ngraph/frontend/onnx_import/op/shape.hpp

```diff
@@ -0,0 +1,49 @@
+//*****************************************************************************
+// Copyright 2017-2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include <memory>
+
+#include "ngraph/node.hpp"
+#include "ngraph/node_vector.hpp"
+#include "ngraph/shape.hpp"
```

Comment:
Please split the implementation into a `.cpp` file, so we don't add too many `includes`

Reply:
There is no need to split file only because of includes because they have `#pragma once` inside.

Reply:
There is a reason to split the `#includes` out to avoid unnecessarily polluting the `ops_bridge` compilation unit's space with all includes.